### PR TITLE
docs: update subscription management available components

### PR DESCRIPTION
**Pull request by vividviolet**

### CODE REVIEW

File: docs/ExtensionPoints/SubscriptionManagement/README.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+render(ExtensionPoint.SubscriptionManagementAdd, Add);
+render(ExtensionPoint.SubscriptionManagementCreate, Create);
+render(ExtensionPoint.SubscriptionManagementEdit, Edit);
+render(ExtensionPoint.SubscriptionManagementRemove, Remove);
+```
+
+### Data API
+
+See [Data](../../Utilities/Data.md) for more info about how to access subscription data provided by the host page.
```
Comment (rpastorelle) [inline docs/ExtensionPoints/SubscriptionManagement/README.md:41]: Data documentation is missing (there are `ProductData` docs)

File: docs/Components/OptionList.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+  root.mount();
+});
+```
+
+#### React
+
+```jsx
+import {ExtensionPoint, OptionList} from '@shopify/argo-admin';
+import {render} from '@shopify/argo-admin/react';
```
Comment (rpastorelle) [inline docs/Components/OptionList.md:39]: Should be imported from `argo-admin-react` (?)

File: docs/ExtensionPoints/SubscriptionManagement/README.md (hunk 1 above)
Comment (vividviolet) [reply docs/ExtensionPoints/SubscriptionManagement/README.md:41]: `ProductData` was changed to `Data` - I've removed it

File: docs/Components/OptionList.md (hunk 2 above)
Comment (vividviolet) [reply docs/Components/OptionList.md]: Fixed!

**Pull request merged**